net_handler/tcp_conn.c

```diff
@@ -332,7 +283,7 @@ static void* tcp_process(void* tcp_args) {
     //variables used for waiting for something to do using select()
     fd_set read_set;
     int log_fd;
-    int maxfd = args->challenge_fd > args->conn_fd ? args->challenge_fd : args->conn_fd;
+    int maxfd = args->conn_fd;
```

Comment:
idk if this makes it easier to understand or not but what do you think about
```
if (args->send_logs) {
    log_fd = fileno(args->log_file);
    maxfd = log_fd > args->conn_fd ? log_fd : conn_fd;
} else {
    maxfd = args->conn_fd;
}
```

Reply:
I think I prefer how its written now.